Source/WebCore/page/LocalDOMWindow.cpp

```diff
@@ -955,7 +955,7 @@ void LocalDOMWindow::processPostMessage(JSC::JSGlobalObject& lexicalGlobalObject
         if (!globalObject)
             return;
 
-        auto& vm = globalObject->vm();
+        Ref vm = globalObject->vm();
```

Comment:
I think the JSC team has expressed concerns about ref'ing the VM unnecessarily?

Reply:
Got it, I removed it.